### DISCUSSION TITLE
Support Ctrl+F in all docks

### DISCUSSION THREAD

**Discussion by passivestar** [godotengine/godot-proposals#8845] [Editor]
Pressing Ctrl+F in the FileSystem dock puts focus into the searchbar, but pressing it in Scene, Inspector and Node docks doesn't do it

**Comment by Calinou** [comment]
This should be relatively easy to do, although we need to be careful to handle focus parents correctly so that shortcuts aren't "stolen" when it's not needed.

Feel free to open a pull request for this :slightly_smiling_face: 

**Edit:** Pull request opened: https://github.com/godotengine/godot/pull/87200
